server/sonar-process-monitor/src/main/java/org/sonar/application/config/JdbcSettings.java

```diff
@@ -110,7 +111,7 @@ Provider resolveProviderAndEnforceNonnullJdbcUrl(Props props) {
   }
 
   private static String buildH2JdbcUrl(int embeddedDatabasePort) {
-    return "jdbc:h2:tcp://localhost:" + embeddedDatabasePort + "/sonar";
+    return "jdbc:h2:tcp://" + InetAddress.getLoopbackAddress().getHostAddress() + ":" + embeddedDatabasePort + "/sonar";
```

Comment:
in fact no, I won't add a dependency on guava for that. However I will fix the issue.